### ISSUE TITLE
Missing permission for AWS Application Load Balancer

### ISSUE THREAD

**Issue by wuttke** [edgelesssys/constellation#3056]
### Issue description

The Control Plane Policy does not have this permission:
elasticloadbalancing:DescribeListenerCertificates

This is required for Ingress that create an ALB with Certicates

**Comment by wuttke** [comment]
```
Annotations:                alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:eu-central-1:905418322627:certificate/658ea803-f7f2-4a4d-bb11-db8e9e480ea8
                            alb.ingress.kubernetes.io/group.name: mesalvo-constellation-ingress-group
                            alb.ingress.kubernetes.io/listen-ports: [{"HTTP": 80}, {"HTTPS": 443}]
                            alb.ingress.kubernetes.io/scheme: internet-facing
                            alb.ingress.kubernetes.io/ssl-redirect: 443
                            alb.ingress.kubernetes.io/success-codes: 200,401
                            alb.ingress.kubernetes.io/target-type: ip
                            meta.helm.sh/release-name: mesalvo-kafka-ui-release-c00d1040
                            meta.helm.sh/release-namespace: kafka
Events:
  Type     Reason             Age   From     Message
  ----     ------             ----  ----     -------
  Warning  FailedDeployModel  54m   ingress  Failed deploy model due to AccessDenied: User: arn:aws:sts::905418322627:assumed-role/mesalvo-cortex-constellation_control_plane_role/i-085b4ad0577f647a8 is not authorized to perform: elasticloadbalancing:DescribeListenerCertificates because no identity-based policy allows the elasticloadbalancing:DescribeListenerCertificates action
           status code: 403, request id: 0131df13-2bc4-4b7d-9c60-5e90bd8923a7
```

**Comment by wuttke** [comment]
That was all I need ;)
  Normal   SuccessfullyReconciled  8m42s  ingress  Successfully reconciled

**Comment by wuttke** [comment]
Thank you very much! Both for the quick reply and the good pointers. This was really helpful.

**Comment by burgerdev** [comment]
The fix is included in [v2.16.4](https://github.com/edgelesssys/constellation/releases/tag/v2.16.4). You can apply the additional permissions with `constellation iam upgrade apply`.
